src/main/java/org/example/youth_be/artwork/enums/ArtworkStatus.java

```diff
@@ -0,0 +1,5 @@
+package org.example.youth_be.artwork.enums;
+
+public enum ArtworkStatus {
+    PUBLIC, SELLING, FREE, PRIVATE
+}
```

Comment:
```suggestion
public enum ArtworkStatus {
    private String description;
    PUBLIC("공개"), SELLING("판매중"), FREE("무료 나눔"), PRIVATE("나만 보기")
}
```

이런식으로 프론트에서 바로 사용할 수 있는 description도 추가하는게 어떨까요?

Reply:
좋습니다. 해당 부분이랑 getname, getdescription 도 만들어 놓을게요